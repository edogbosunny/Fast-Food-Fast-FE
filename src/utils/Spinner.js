import React from 'react';
import spinnerImage from '../img/spinner.gif';


export default function Spinner() {
  return (
    <div>
      <img
        style={{ width: '200px', margin: 'auto', display: 'block' }}
        src={spinnerImage}
        alt='LOADING...'
      />
    </div>
  );
}
