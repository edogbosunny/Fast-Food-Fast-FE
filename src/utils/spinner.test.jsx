import React from 'react';
import { shallow } from 'enzyme';
import Spinner from './Spinner';

describe('Spinner Test', () => {
  const wrapper = shallow(<Spinner />);

  it('should render spinner', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
