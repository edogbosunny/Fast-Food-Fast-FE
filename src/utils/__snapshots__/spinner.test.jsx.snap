// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Test should render spinner 1`] = `
<div>
  <img
    alt="LOADING..."
    src="test-file-stub"
    style={
      Object {
        "display": "block",
        "margin": "auto",
        "width": "200px",
      }
    }
  />
</div>
`;
